nimble/host/audio/services/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -286,6 +286,9 @@ struct ble_svc_audio_bass_operation {
             /** Number of subgroups */
             uint8_t num_subgroups;
 
+            /** BIS Synchronisation of subgroups */
+            uint32_t bis_sync[BLE_SVC_AUDIO_BASS_SUB_NUM_MAX]
+
```

Comment:
This seems to me not needed, as the bis_sync is passed in `struct ble_svc_audio_bass_subgroup.bis_sync_state` (1).

Reply:
Removed